Common/Securities/UniverseManager.cs

```diff
@@ -35,6 +36,14 @@ public class UniverseManager : IDictionary<Symbol, Universe>, INotifyCollectionC
         /// </summary>
         public event NotifyCollectionChangedEventHandler CollectionChanged;
 
+        /// <summary>
+        /// Read-only dictionary containing all active securities. An active security is
+        /// a security that is currently selected by the universe or has holdings or open orders.
+        /// </summary>
+        public IReadOnlyDictionary<Symbol, Security> ActiveSecurities => this
+            .SelectMany(ukvp => ukvp.Value.Members.Select(mkvp => mkvp.Value))
+            .DistinctBy(s => s.Symbol).ToDictionary(s => s.Symbol);
```

Comment:
We *could* manage a direct instance via intercepting the events of when securities are added/removed from a universe but it seemed far too complex and a premature optimization. Since this collection is read-only, no unexpected behavior should arise from receiving a new instance every time you access the property.